website_snippet_country_dropdown/views/snippets.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<openerp>
+<data>
+
+<template id="website_snippet_country_dropdown"
+          inherit_id="website.snippets"
+          name="Country Dropdown">
+    <xpath expr="//div[@id='snippet_content']">
+        <div>
+
+            <div class="oe_snippet_thumbnail">
+                <img src="/website_snippet_country_dropdown/static/src/img/dropdown.png"
+                     class="oe_snippet_thumbnail_img"/>
+                <span class="oe_snippet_thumbnail_title">
+                    Country Dropdown
+                </span>
+            </div>
+
+            <div class="oe_snippet_body website_snippet_country_dropdown js_country_dropdown">
+                <t t-call="website_snippet_country_dropdown.dropdown_element_form"/>
+            </div>
+
+        </div>
+    </xpath>
+    <xpath expr="//div[@id='snippet_options']" position="inside">
+        <div data-snippet-option-id='website_snippet_country_dropdown'
+            data-selector=".js_country_dropdown"
+            data-selector-siblings=""
+            data-selector-children="form">
+            <li>
+                <a href="#" class="button js_dropdown_fields">Set input options</a>
+            </li>
+
+        </div>
+    </xpath>
+</template>
+
+<template id="dropdown_element_form">
+    <div class="form-group" t-ignore="true">
+        <label class="col-md-3 col-sm-4 control-label"
+               for="contact_name">
+            <span t-if="label_country_dropdown" t-esc="label_country_dropdown"/>
+            <span t-if="not label_country_dropdown">Your Label</span>
+        </label>
+        <div class="col-md-7 col-sm-8">
+            <t t-call="website_snippet_country_dropdown.country_dropdown"/>
+        </div>
+    </div>
+</template>
+
+<template id="country_dropdown" name="Country Dropdown">
+    <t t-set="countries" t-value="request.env['res.country'].search([])"/>
+    <div id="snippet_container" class="input-group" t-ignore="true">
+        <div class="input-group-btn">
+            <button type="button" id="btn_vat_code"
+                    class="btn btn-default dropdown-toggle js_btn_country_code form-control"
+                    data-toggle="dropdown" t-att-data-country_code="countries[:1].code">
+                <img id="img_code_vat" class="js_img_country_code"
+                     t-attf-src="/website/image/res.country/#{country_vat_code_id or 1}/image/30x20"
```

Comment:
If I understand correctly, `country_vat_code_id` sets the default country ID that will be selected.

If so, I'd rename it to `country_default_id`, add this setting to the README, and possibly default to the user's, website's or website company's country (the best one available).